python/cugraph/traversal/sssp_wrapper.pyx

```diff
@@ -32,77 +33,112 @@ import cudf._lib as libcudf
+                                                     num_verts,
+                                                     num_edges)
+            graph_float.get_vertex_identifiers(<int*> c_identifier_ptr)
+            c_sssp.sssp[int, int, float](graph_float,
+                                         <float*> c_distance_ptr,
+                                         <int*> c_predecessor_ptr,
+                                         <int> source)
+        elif data_type == np.float64:
+            graph_double = GraphCSR[int, int, double](<int*> c_offsets_ptr,
+                                                      <int*> c_indices_ptr,
+                                                      <double*> c_weights_ptr,
+                                                      num_verts,
+                                                      num_edges)
+            graph_double.get_vertex_identifiers(<int*> c_identifier_ptr)
+            c_sssp.sssp[int, int, double](graph_double,
+                                          <double*> c_distance_ptr,
+                                          <int*> c_predecessor_ptr,
+                                          <int> source)
+        else: # This case should not happen
+            raise NotImplementedError
     else:
-        c_bfs.bfs[int](g, <int*>c_distance_ptr, <int*>c_predecessors_ptr, <int>source)
-
+        # TODO: Something might be done here considering WT = float
+        graph_float = GraphCSR[int, int, float](<int*> c_offsets_ptr,
+                                                <int*> c_indices_ptr,
+                                                <float*> NULL,
+                                                num_verts,
+                                                num_edges)
+        graph_float.get_vertex_identifiers(<int*> c_identifier_ptr)
+        c_bfs.bfs[int, int, float](graph_float,
+                                   <int*> c_distance_ptr,
+                                   <int*> c_predecessor_ptr,
+                                   <int> source)
+
+    # Step 9: Renumber before return
+    #         It is only required to renumber vertex and predecessors
     if input_graph.renumbered:
-        if isinstance(input_graph.edgelist.renumber_map, cudf.DataFrame):
-            n_cols = len(input_graph.edgelist.renumber_map.columns) - 1
-            unrenumbered_df_ = df.merge(input_graph.edgelist.renumber_map, left_on='vertex', right_on='id', how='left').drop(['id', 'vertex'])
-            unrenumbered_df = unrenumbered_df_.merge(input_graph.edgelist.renumber_map, left_on='predecessor', right_on='id', how='left').drop(['id', 'predecessor'])
-            unrenumbered_df.columns = ['distance']+['vertex_'+str(i) for i in range(n_cols)]+['predecessor_'+str(i) for i in range(n_cols)]
-            cols = unrenumbered_df.columns.to_list()
-            df = unrenumbered_df[cols[1:n_cols+1] + [cols[0]] + cols[n_cols:]]
```

Comment:
Is this still supposed to be handled?

Reply:
yes, that looks like the unrenumbering step

Reply:
Reverted it in https://github.com/rapidsai/cugraph/pull/764/commits/1528aebe9ea39d6ac0060b1068a76f8c2a36056c